### PR TITLE
added a loading indicator to improve responsiveness

### PR DESCRIPTION
**Pull request by linaya2004**
I made changes in these files:
custom_static/css/hardhat.css:  layout styles for responsiveness
custom_static/js/load_more.js:  functionality for loading indicators

### CODE REVIEW

File: home/templates/pages/search-results.html [hunk 1]
```diff
@@ -51,6 +51,7 @@
 
   {% block content %}
     <div class="container">
+    <div id="loading-indicator" style="display:none;">Loading...</div>
```
Comment (s-bonnor) [inline home/templates/pages/search-results.html:54]: So when this is visible, which currently it never is, its at the top of the page under the navigation bar
could you reposition it so when it is visible its correctly positions

File: custom_static/js/load_more.js
Comment (s-bonnor) [inline custom_static/js/load_more.js:1]: This file isn't actually used by the search results page, its used by the currently mostly broken blog index page

**Review by s-bonnor — CHANGES_REQUESTED** [review]
Please fix the positioning of the Loading Indicator

File: custom_static/js/load_more.js
Comment (linaya2004) [reply custom_static/js/load_more.js:1]: sorry please disregard the changes on the load_more.js page. i will remove it

File: home/templates/pages/search-results.html (hunk 1 above)
Comment (linaya2004) [reply home/templates/pages/search-results.html:54]: ill fix it, thank you

**Pull request closed without merging**